app/page.tsx

```diff
@@ -281,15 +282,15 @@ export default function Home() {
 
         <SectionContent id='latest_research'>
           <SectionTitle>Latest Research</SectionTitle>
-            <div className="col-span-12 lg:col-span-4">
-            <Link href="/projects/agent_vs_script">
+          <div className="col-span-12 lg:col-span-4">
+            <Link href="/projects/sotopia-rl">
               <ImageWithTextOverlay
-              src={agentvsstory}
```

Comment:
@lwaekfjlk instead of removing existing works, any chance we could have a scrolling bar that still allows viewers to check older works?